docs/conf.py

```diff
@@ -36,6 +37,7 @@
               'sphinx.ext.viewcode',
               'jupyter_sphinx',
               'reno.sphinxext',
+              'qiskit_sphinx_theme',
              ]
 html_static_path = ['_static']
 templates_path = ['_templates']
```

Comment:
You should get rid of these two lines and the next one, and also delete `style.css`. There's no need to override qiskit_sphinx_theme. The only time people really do is adding custom autodoc summaries.

Reply:
Okay I've just removed these. For the autodoc summaries, we'll be wanting to use these as we start writing the docs, should I copy over the `docs/_templates` from the Dynamics repo to add these?

Reply:
We don't have a standard format for autodoc, but Dynamics is probably reasonable, or Terra.